kotlin/workflow-ui/core-compose/api/core-compose.api

```diff
@@ -1,5 +1,5 @@
 public final class com/squareup/workflow/ui/compose/ComposeViewBinding : com/squareup/workflow/ui/ViewBinding {
-	public fun <init> (Lkotlin/reflect/KClass;Lkotlin/jvm/functions/Function2;)V
+	public fun <init> (Lkotlin/reflect/KClass;Lkotlin/jvm/functions/Function3;)V
```

Comment:
API update required since dev06 starts adding a synthetic composer parameter to all composable functions.